lib/src/linting/rules/no-nested-types-within-unions.ts

```diff
@@ -16,6 +17,7 @@ export const noNestedTypesWithinUnions: LintingRule = contract => {
     topLevelTypes.map(t => extractNestedUnionTypes(t.type, t.name))
   );
   return unionTypes
+    .filter(negate(isUnionOfSingleTypeWithNull))
```

Comment:
Does this mean that we want to 'catch' types that do not have a union of single | null? Or am I understanding this the other way around

Reply:
This is saying "exclude types that are of the form SingleType | null` :)

Reply:
oooooooohh, yeah the other way around.